src/api.v2/controllers/experimentController.js

```diff
@@ -31,11 +44,47 @@ const createExperiment = async (req, res) => {
     ),
   ]);
 
-  logger.log('Finished creating experiment');
+  logger.log(`Finished creating experiment ${result.id}`);
+
+  res.json(OK());
+};
+
+const patchExperiment = async (req, res) => {
+  const { params: { experimentId }, body } = req;
+
+  logger.log(`Updating experiment ${experimentId}`);
+
+  const snakeCasedKeysToPatch = _.mapKeys(body, (_value, key) => _.snakeCase(key));
```

Comment:
this will probably be used in other places too. Why don't you move it in `helpers` next to `sqlToCamelCased`?

Reply:
I think it is not very sql-specific though, it is a very generic function compared to `sqlToCamelCased` so I think it fits more in utils

Reply:
perhaps we could leave it here and abstract it when we see how it can be reused?

Reply:
sure, we can leave it here for now